superset-frontend/src/dashboard/actions/hydrate.js

```diff
@@ -394,6 +394,11 @@ export const hydrateDashboard =
 
     const { roles } = user;
     const canEdit = canUserEditDashboard(dashboard, user);
+    const crossFiltersEnabled =
+      (isFeatureEnabled(FeatureFlag.DASHBOARD_CROSS_FILTERS) &&
+        (metadata.cross_filters_enabled === undefined ||
```

Comment:
Nit: We have this logic in 2 places (true if undefined), maybe we could move it to a utility function?